pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -175,27 +176,36 @@ func (f *FakeDockerClient) InspectImage(name string) (*docker.Image, error) {
 	return f.Image, err
 }
 
+func normalSleep(mean, stddev, cutoffMillis int) {
```

Comment:
Could you add an sentence here explaining how this function should be used? The unit of the variables is also unclear.